.github/workflows/storybook.yml

```diff
@@ -0,0 +1,38 @@
+# Workflow name
+name: 'Chromatic Deployment'
+
+on:
+  pull_request:
+    branches: [main]
+  push:
+    branches: [main]
+
+jobs:
+  chromatic-deployment:
+    runs-on: ubuntu-latest
+    steps:
+      - name: actions/checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Set node linker to nodeModules
+        run: yarn config set nodeLinker "node-modules"
```

Comment:
넵!!! 찾아볼게요!!! Github actions에 대해 자세한 지식이 없어서 레퍼런스 잡고 코드 사용했는데 더 알아보겠습니다 🥲 !!

Reply:
이거 한 번 해볼래요?? [Automate Chromatic with GitHub Actions](https://www.chromatic.com/docs/github-actions/#workflow-setup)

아래 부분 빼고 버전도 최신으로 변경해서 한 번 해보고 아니되면 걍 완벽 천재 도은님 코드로 반영 ㄱㄱ하시져~!! 
``` yml
      - name: Set node linker to nodeModules
        run: yarn config set nodeLinker "node-modules"
```


** 주의 ** (문서의 `--immutable-cache` 옵션은 넣으면 안돼요~ 패키지 내에 모든 cache가 잇는게 아니라 변경 막으면 안됨)

옵션 설명:
`--immutable`: This option ensures that the installation process is immutable, meaning it doesn't change the yarn.lock file. If an update is needed, the command will fail instead of updating the lock file. This is useful in continuous integration environments where you want to ensure that the lock file is up-to-date and any needed updates are done deliberately.

`--immutable-cache`: This option prevents Yarn from updating its global cache. If a package needed for the project isn't already in the cache, the command will fail instead of updating the cache. This is also useful in continuous integration environments where you want to ensure that the cache is up-to-date.

`--check-cache`: This option instructs Yarn to always refetch the packages and to ensure their checksums match what's listed in the lock file. This can help detect corrupted downloads and ensure the integrity of your dependencies.

Reply:
@deepbig 오 대박 갓홍석 킹홍석 배포 성공됐어요!! 
기분탓인지 모르겠는데 좀 더 빨라진 기분 ~ 굳굳 감삼다 ,, 🕺🕺🕺

Reply:
오오오오~! 최고최고~!! 아주 조금 (20초) 빨라졌네요~!ㅋㅋㅋㅋㅋ 프로젝트 사이즈가 더 커지면 더 많이 빨라질 거라 믿어보죠~!! 수고 많았어요 도은 동생~!! 👍 💯 👍 💯 👍 💯 👍 💯 
![image](https://github.com/depromeet/amazing3-fe/assets/34956359/d03ca6ae-d057-400a-adf8-353d5478d8d1)